Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -62,6 +61,9 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin)
 
   for (size_t i = 0; i < NAND_TOTAL_BLOCKS; i++)
   {
+    if (i % 1000 == 0)
```

Comment:
No reason in particular. All I needed was to make the progress bar update once in a while, so I found 1000 cycles was a good number between not updating at all and updating all the time.

If there is a better way to update at a set interval, please let me know.